src/webview/components/memory-widget.tsx

```diff
@@ -32,14 +33,16 @@ interface MemoryWidgetProps extends MemoryDisplayConfiguration {
     effectiveAddressLength: number;
     isMemoryFetching: boolean;
     refreshMemory: () => void;
-    updateMemoryArguments: (memoryArguments: Partial<DebugProtocol.ReadMemoryArguments>) => void;
+    updateMemoryArguments: (memoryArguments?: MemoryOptions) => void;
     toggleColumn(id: string, active: boolean): void;
     isFrozen: boolean;
     toggleFrozen: () => void;
     updateMemoryDisplayConfiguration: (memoryArguments: Partial<MemoryDisplayConfiguration>) => void;
     resetMemoryDisplayConfiguration: () => void;
     updateTitle: (title: string) => void;
-    fetchMemory(partialOptions?: Partial<DebugProtocol.ReadMemoryArguments>): Promise<void>
+    fetchMemory(partialOptions?: MemoryOptions): Promise<void>;
+    triggerStoreMemory(): void;
+    triggerApplyMemory(): void;
```

Comment:
All function calls trigger some action:

```suggestion
    storeMemory(): void;
    applyMemory(): void;
```